### PR TITLE
Odin Sign-Up Form

### PR DESCRIPTION
**Pull request by Sushilkumar3255**
Description:
This PR introduces the Sign up form of the Odin Project's Intermediate HTML and CSS section. From this project I've implemented the use of forms and their action.

Features:
The forms input fields are required such that It can't be submitted before filling them out correctly.
Placeholder are placed such that It can give idea of filling it.
Added Box shadow on the Form area so that it will be differentiated from other sections
Added input box Focus property so that when it is selected it is highlighted with different color.
The Create Account button does not collect any information for now.

Live Link: https://odin-signup-form.netlify.app/

Project Link - https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-sign-up-form

UI Screenshot:
odin sign-up

### CODE REVIEW

File: Odin-Sign-Up-Form.zip
Comment (roopaljasnani) [inline Odin-Sign-Up-Form.zip:1]: I don't think this Zip is needed, pls remove

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odin Sign-Up Form</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="main.js"></script>
```
Comment (roopaljasnani) [inline index.html]: self-closing tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    <br />
+                    like this in your life to help you realize your deepest dreams.
+                    <br />
+                    Sign up <em>now</em> to get started. <br /><br />
+                    You <em>know</em> you want to.
+                </h3>
+            </div>
+            <form action="" method="get">
+                <div class="second">
+                    <h3 style="margin-left: 50px; padding-top: 20px;">Let's do this!</h3>
```
Comment (roopaljasnani) [inline index.html]: no inline styles pls

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    <div class="form-main-div">
+                        <div class="form-divs">
+                            <div class="form-div">
+                                <label for="firstName">FIRST NAME</label>
+                                <input type="text" name="firstName" id="firstName" placeholder="John" required>
```
Comment (roopaljasnani) [inline index.html]: inputs also have to self-closing

Comment (roopaljasnani) [inline index.html]: you can think of adding constraints on input like minLength , pattern etc.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    color: white;
+    font-size: small;
+    font-weight: bolder;
+    height: 40px;
+    width: 175px;
+}
+
+.third,.hyperlink {
+    margin-left: 50px;
+    margin-top: 25px;
```
Comment (roopaljasnani) [inline style.css]: shorthand margin

File: Odin-Sign-Up-Form.zip
Comment (Sushilkumar3255) [reply Odin-Sign-Up-Form.zip:1]: Done. Mam

File: index.html (hunk 1 above)
Comment (Sushilkumar3255) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (Sushilkumar3255) [reply index.html]: Okay.

File: index.html (hunk 3 above)
Comment (Sushilkumar3255) [reply index.html]: Done.

Comment (Sushilkumar3255) [reply index.html]: okay.

File: style.css (hunk 4 above)
Comment (Sushilkumar3255) [reply style.css]: Done.

**Review by roopaljasnani — APPROVED** [review]

**Pull request merged**